app/components/Button/Button.test.js

```diff
@@ -25,9 +25,9 @@ describe('<Button />', () => {
     expect(renderedComponent.find('a').length).toEqual(1);
   });
 
-  it('should render a Link if the route prop is specified', () => {
-    const renderedComponent = shallow(<Button route="/" />);
-    expect(renderedComponent.find('Link').length).toEqual(1);
+  it('should render a button to change route if the handleRoute prop is specified', () => {
+    const renderedComponent = shallow(<Button handleRoute={ ()=> {} } />);
```

Comment:
When I used `shallow` method ,  `handleRoute={ ()=> {} }` and `handleRoute='anything'` both passed the test.  Did shallow rendering ignore the propTypes ?
```js
Button.propTypes = {
  handleRoute: React.PropTypes.func,
};
```